extensions/interactions/base.py

```diff
@@ -91,6 +91,11 @@ class BaseInteraction(object):
     # Customization arg specifications for the component, including their
     # descriptions, schemas and default values. Overridden in subclasses.
     _customization_arg_specs = []
+    # Instructions for using this interaction, to be shown to the learner. Only
+    # relevant for supplemental interactions.
+    instructions = None
+    # Whether the answer is long, and would benefit from being summarized.
+    needs_summary = False
```

Comment:
This is interesting. My first inclination was to regard it as coincidental -- but @amitdeutsch, any thoughts? (Note that the summary displays a shorter form of the answer, and the full answer is displayed as a popover on-hover.)